src/test/resources/db/migration/R__test_data.sql

```diff
@@ -26,7 +26,29 @@ VALUES ('7fffcc6a-11f8-4713-be35-cf5ff1aee517', 'd3abc217-75ee-46e9-a010-368f302
 
        ('b328ebc8-1f7b-4236-b4ac-30f50b43a92d', '1811faa6-d568-4fc4-83ce-41118b90242e', 'BWN', 'nobody-bwn@digital.justice.gov.uk', null);
 
+INSERT INTO course_participation (course_participation_id, prison_number, course_name, detail)
+VALUES ('53787979-c455-4a3f-9787-9cd2f69a9dad', 'E6NInAW', 'Grey Course', 'Textor atrox accommodo corona volva conturbo suscipit civis.'),
+       ('1811faa6-d568-4fc4-83ce-41118b90242e', 'TPPIdBJ', 'Black Course', 'Men who deny their current offence can participate in BBR');
+
+
+--
+-- val id: UUID? = null,
```

Comment:
Try not to leave comments lying around unless they're absolutely necessary.

Reply:
I think you have reviewed an outdated file.